### ISSUE TITLE
Integration of chromosomal segments lists for annotation and filtering

### ISSUE THREAD

**Issue by TJO-Lund** [Clinical-Genomics/scout#1907]
Integration of lists with information about chromosomal segments (including several genes e.g., ”chr22: 19,037,148 - 21,228,744”) for use in Scout in the same manner as we use gene lists. This is essential for better/correct evaluation of SV-variants such as microdeletion and microduplication syndromes. For example we would like to use ClinGen Regions (https://dosage.clinicalgenome.org/pathogenic_region.shtml) for both annotating purposes and filtering. It should also be easy to add similar, in house, segment lists to Scout. Can this be a prioritized issue? In communication with Daniel Nilsson; Tord Jonson Clinical Genomics Lund

**Comment by dnil** [comment]
Thanks! We have discussed this many times (and there are probably some issues somewhere relating to it in conjunction with a future panel revamp, where we would also like to support repeat expansions).

A more near term workaround would be to allow reading coordinate files and construct gene symbol lists from the genes overlapping them. This will have a slightly different end result, but variants without genes on them are not terribly likely to appear ranked to a level where the operator sees them anyway. Do you have any vital counterexamples where completely intergenic events are established disease causing?

**Comment by TJO-Lund** [comment]
One axample is this deletion "SHOX Deficiency Disorders, including Leri-Weill dyschondrostosis - critical regulatory element"* (chrX:819,816-867,357) - we have had one such finding with genomic array. Just settling with gene-lists is, IMO, simply suboptimal and I strongly believe that it will be neccessary to work with regions as well as with gene-lists, as our understanding of our genome evolves. Even today it is of paramount importance not to miss atypical deletions (when working with narrow gene lists, such as a cardio-gene list), which occur with many syndromes, for example 22q11 deletion syndrome with other breakpoints than the typical A-D variant. See (https://dosage.clinicalgenome.org/pathogenic_region.shtml).

*Extract from our local database Mimisbrunnr states "Region as defined in PMID: 22791839 "Identification of the first recurrent PAR1 deletion in Léri-Weill dyschondrosteosis and idiopathic short stature reveals the presence of a novel SHOX enhancer" and in PMID: 23636926 (2013) "Diagnostic screening identifies a wide range of mutations involving the SHOX gene, including a common 47.5 kb deletion 160 kb downstream with a variable phenotypic effect".
See also Decipher https://decipher.sanger.ac.uk/syndrome/58#overview; OMIM #127300 (heterozygous loss), #249700 (homozygous loss, Langer mesomelic dysplasia) and 300582 (Short stature, idiopathic familial); ORPHA 240 (Léri-Weill dyschondrosteosis), 2632 (Langer mesomelic dysplasia) and 314795 (SHOX-related short stature); GeneReviews PMID: 20301394 "SHOX Deficiency Disorders"; and ClinGen Gene Curation: www.ncbi.nlm.nih.gov/projects/dbvar/clingen/clingen_gene.cgi?sym=SHOX&subject="

**Comment by TJO-Lund** [comment]
Just an idea from Hans, is it at all possible to include regions and STRs  in a single list using "tags" for regions, genes or STRs? That would perhaps make it easier to apply a single list when filtering for different indications.

**Comment by TJO-Lund** [comment]
Does "Hard" mean that we should give up on this important enhancement or does it inspire someone to reach a difficult goal?  :-)

**Comment by dnil** [comment]
You tell me! 😄

**Comment by dnil** [comment]
No, but seriously, we have not given it up. It will happen. If you'd like to contribute, that is very welcome, but otherwise it is also good also to know that you consider it important. That SHOX enhancer is a good example. We have some issues with it on the vcf2cytosure track as well - those don't look terribly clean in the PAR-regions - and quite potentially in calling as well.

**Comment by TJO-Lund** [comment]
Great! 
I'd love to help out any way I can, and you may include me if you set up a group that will work with this issue.

**Comment by 4WGH** [comment]
We also need to have specific region in our panel, not always we can use a gene(s) with the risk for incidental findings in the SNV page. In panelapp there are ISCAs region with coodinates thast we would like to analyze in the SV view. 
an exemple
https://panelapp.genomicsengland.co.uk/panels/509/region/ISCA-37393-Gain/#!details.
could be good to ba able to use them :)

**Comment by TJO-Lund** [comment]
3 years+ for this issue, any recent updates/progress?

**Comment by dnil** [comment]
Oh, the question is much older. 😉 Suggestions are welcome!
